frontend/src/scenes/teamLogic.ts

```diff
@@ -136,6 +141,12 @@ export const teamLogic = kea<teamLogicType>({
             if (contextualTeam) {
                 // If app context is available (it should be practically always) we can immediately know currentTeam
                 actions.loadCurrentTeamSuccess(contextualTeam)
+                if (contextualTeam.ingested_event) {
+                    const builtEventUsageLogic = eventUsageLogic.build()
+                    const unmount = eventUsageLogic.mount()
+                    builtEventUsageLogic.actions.reportTeamHasIngestedEvents()
```

Comment:
If you add this as a listener to `loadCurrentTeamSuccess`, you wouldn't need to put this report in 2 places